aggregator_core/src/task.rs

```diff
@@ -1088,14 +1082,8 @@ pub mod test_util {
         pub fn new(query_type: QueryType, vdaf: VdafInstance) -> Self {
             let task_id = random();
 
-            let leader_hpke_keypairs = [
-                generate_test_hpke_config_and_private_key(),
-                generate_test_hpke_config_and_private_key_with_id(1),
-            ];
-            let helper_hpke_keypairs = [
-                generate_test_hpke_config_and_private_key(),
-                generate_test_hpke_config_and_private_key_with_id(1),
-            ];
+            let leader_hpke_keypairs = [HpkeKeypair::test(), HpkeKeypair::test_with_id(1)];
+            let helper_hpke_keypairs = [HpkeKeypair::test(), HpkeKeypair::test_with_id(1)];
```

Comment:
I will fix this flake in another PR. I'm surprised this has never flaked on me before.